easy_my_coop/models/coop.py

```diff
@@ -89,8 +89,9 @@ def create(self, vals):
             cooperator.write({"cooperator": True})
         subscr_request = super(SubscriptionRequest, self).create(vals)
 
-        mail_template_notif = subscr_request.get_mail_template_notif(False)
-        mail_template_notif.send_mail(subscr_request.id)
+        if self._send_confirmation_email():
+            mail_template_notif = subscr_request.get_mail_template_notif(False)
```

Comment:
We should name the argument for clarity here and later (I know it's not your code :p):

```py
get_mail_template_notif(is_company=False)
```

Reply:
+1

Reply:
Done!

I change all the calls to the `get_mail_template_notif` method to notify the argument names.